app/src/api/api.ts

```diff
@@ -0,0 +1,141 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import { authUser, createUser, getUserById, getNewUserAPIKey } from './services/user';
+import { queryLocation } from './services/search';
+
+import buildingsRouter from './routes/buildingsRouter';
+
+
+const server = express();
```

Comment:
```suggestion
const server = express.Router()
```

Reply:
This was kind of intentional as I feel long term we should gravitate towards the web app backend and the data API being two separate applications (note that the use of `body-parser` was moved to only be applied in `api.ts` as we don't use JSON requests in the frontend serving part of the app), but actually since we don't start them separately, a Router is probably fine for now.

Reply:
Makes sense as a design idea - at some point back in time I had the front end, tileserver and API server split out as separate, but YAGNI and deployment simplicity won out, especially before we have data on performance and usage stats - if any request can be handled by one kind of process then we can simply scale that process and load should be spread simply across them.